jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -589,6 +589,28 @@ class Status extends React.Component {
   }
 
   /**
+    Generate a link to an application from a relation.
+    @param name {String} An app name.
+    @returns {Object} The link element.
+  */
+  _generateRelationAppLink(name) {
+    const app = this.props.db.services.getById(name);
+    if (!app) {
+      // If the application is not in the DB it must be remote app so don't
```

Comment:
I don't think this is a reliable/future proof way of doing this as I could see a time when we'd want all applications in the database for easy querying of information. Was there not something unique with the id's that we could trigger instead?

Reply:
Sorry you can disregard this comment, I forgot that we store the remoteApplications in the db in another object.